pyproject.toml

```diff
@@ -26,7 +26,8 @@ classifiers = [
 ]
 dependencies = [
     "Sphinx>=4.2.0",
-    "pydata-sphinx-theme>=0.14.0",
+    "pydata-sphinx-theme@git+https://github.com/pydata/pydata-sphinx-theme#egg=main",
```

Comment:
The issue with this is that using a branch does not allow to upload a new version to PyPI.

We need to limit the upper version for the `pydata-sphinx-theme`.